deploy/static/static.yaml

```diff
@@ -21,25 +21,44 @@ spec:
     spec:
       containers:
         - name: from-file-prometheus-exporter
-          image: python:alpine3.9
-          env:
+          image: golang:alpine
+          env: # Files will be downloaded from the following URLs if they are not empty
             - name: METRICS_FILE_URL
-              value: "<ADD_YOUR_FILE_URL_HERE>" # You can use private gist url.
+              value: "https://raw.githubusercontent.com/newrelic/nri-prometheus/santalla/mockexporter/internal/mockexporter/metrics.file"
+            - name: TESTER_SCRIPT_URL
+              value: "https://raw.githubusercontent.com/newrelic/nri-prometheus/santalla/mockexporter/internal/mockexporter/loadtest.go"
```

Comment:
Now I get what you meant during the daily, it is the first time I see this approach 😄 
I prefer the Dockerfile to build and image and use that locally, but if it is works let's keep this :)

Reply:
Dockerfile is up now as well!